include/dlaf/matrix/distribution.h

```diff
@@ -282,6 +283,21 @@ class Distribution {
                                                      source_rank_index_.get<rc>());
   }
 
+  /// Returns the local index in process @p rank of global tile
+  /// whose index is the biggest index smaller or equal @p global_tile
+  /// and which is stored in process @p rank. If there is no such tile
+  /// index, -1 is returned.
+  ///
+  /// @pre 0 <= global_tile <= nrTiles().get<rc>().
+  template <Coord rc>
+  SizeType prevLocalTileFromGlobalTile(SizeType global_tile) const noexcept {
+    DLAF_ASSERT_HEAVY(0 <= global_tile && global_tile <= global_nr_tiles_.get<rc>(), global_tile,
+                      global_nr_tiles_.get<rc>());
+    return util::matrix::prevLocalTileFromGlobalTile(global_tile, grid_size_.get<rc>(),
+                                                     rank_index_.get<rc>(),
+                                                     source_rank_index_.get<rc>());
+  }
```

Comment:
Might be not needed. See `localSizeFromGlobalTileIndexRange` comment.

Reply:
Removed here : https://github.com/eth-cscs/DLA-Future/pull/676/commits/b72c403a453739c38cdd97d778259c137605b5c2